README.md

```diff
@@ -1 +1,35 @@
-# Leader-board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leaderboard is a webapp where user can maintain a list of anmes and scores in a game.
+
+<img width="563" src="./src/Leaderboard.png">
+
+
+## Built With
+
+- HTML
+- Css
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://chukwuebukavictor.github.io/Leader-board/dist)
+
+## Getting Started
+
+**Follow the steps below to run this project on you local machine**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Understanding of git and github
+### Setup & Install
+- Clone repo on your local device using `git clone <link>` command
```

Comment:
- [ ] Kindly improve the README to make it more descriptive by adding instructions on how to clone your repository to the terminal, installing your dependencies by running `npm install`, starting your app to be viewed in the browser by running `npm run start` and building your app by running `npm run build`. This will make it easier for a user to get your app up and running in the terminal 👍.

- [ ] Kindly add the `contributing`, `show your support` and `acknowledgment` sections from this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to your README as it is required to have all the necessary sections from the template.